packages/phasor/src/renderer.ts

```diff
@@ -69,6 +72,10 @@ export class Renderer implements SurfaceViewport {
     this.ctx = this.canvas.getContext('2d') as CanvasRenderingContext2D;
     this.rc = new RoughCanvas(canvas);
   }
+  clientToModelCoord(vec: IVec): IVec {
```

Comment:
This method lacks consistency with existing ones (`toViewCoord` / `toModelCoord`). Having the caller side compute the bounding rect makes more sense. Please DO care about consistency, otherwise you'll break the codebase in a very fast way, which is NOT acceptable as our official team member.

Also, notice that DO NOT write newly added methods in such arbitrary position, they should be placed relative to the ones with close semantics.